daemon/src/wallet.rs

```diff
@@ -131,6 +135,36 @@ impl Actor {
 
 #[xtra_productivity]
 impl Actor {
+    pub fn handle_recover(&mut self, msg: Recover, ctx: &mut xtra::Context<Self>) -> Result<()> {
+        let client = bdk::electrum_client::Client::new(&self.electrum_rpc_url)
+            .context("Failed to initialize Electrum RPC client")?;
+
+        let seed = Seed::from_seed_words(&msg.seed_words)?;
+
+        let ext_priv_key = seed.derive_extended_priv_key(self.wallet.network())?;
```

Comment:
This is not ideal. I think for now, we should be creating the `ExtendedPrivKey` directly from the seed words, not roundtripping through `Seed` because that also derives the network identity. Let's keep this change isolated to the wallet for now otherwise you would have to re-initialize the connection here as well because on next startup (if you were to use the new seed) you would suddenly have a different identity.